content/docs/main/resiliency/timeouts/idle-stream.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Idle stream timeouts
+weight: 25
+description:
+---
+
+Customize the default idle stream timeout of 5 minutes (300s). 
+
+## About idle stream timeouts
+
+By default, Envoy closes all idle request and response streams after 5 minutes if no data is sent or received and returns a 408 Request Timeout HTTP response code. Idle stream timeouts are different from request timeouts. Request timeouts configure the time Envoy allows for the entire request stream to be received from the client. An idle stream timeout on the other hand is the time Envoy allows a stream to exist without activity before it is terminated. Idle stream timeouts are recommended to protect against clients that stall or that open the stream and never send any data.
+
+You can change the default idle stream timeout setting with a {{< reuse "docs/snippets/trafficpolicy.md" >}}.  While idle streams are a concept in the HTTP/2 and HTTP/3 protocols, Envoy also maps an HTTP/1 request to a stream. Because of that, you can apply idle stream timeouts to HTTP/1 traffic too. 
+
+## Before you begin
+
+{{< reuse "docs/snippets/prereq.md" >}}
+
+## Set up idle stream timeouts
+
+1. Create a {{< reuse "docs/snippets/trafficpolicy.md" >}} with your idle stream timeout settings. If you have an HTTPRoute with multiple HTTPRoute rules, you can use the `targetRefs.sectionName` to apply the timeout to a specific HTTPRoute rule. In this example, you apply the policy to the `timeout` rule that points to the `/headers` path in your HTTPRoute resource.
```

Comment:
> In this example, you apply the policy to the `timeout` rule that points to the `/headers` path in your HTTPRoute resource.

I am not sure the example is doing this? not sure which needs to change, looks like it's just targeting the entire httpbin HTTPRoute.

Reply:
Reading the next step, I am thinking this TrafficPolicy is supposed to target the entire Gateway?

Reply:
this timeout only works per route, the idle one is for the gateway